### PR TITLE
Api branch PR

### PR DESCRIPTION
**Pull request by mertkantaroglu**
Hello,

I made the required changes in the file which are:

- [ ] Used API to fetch and post data.
- [ ] Implemented Refresh button function to get scores
- [ ] Implemented form to submit new score
- [ ] Used async / await
- [ ] Used ES6 and arrow functions

Looking forward to your response.

Best,
Mert

### CODE REVIEW

File: src/modules/showScore.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  data.sort((a, b) => b.score - a.score);
+  listContainer.innerHTML = '';
+  const tableData = data.map((item) => `<tr> <td id="table-user">${item.user}</td> <td id="table-score">${item.score}</td> </tr>`);
+  listContainer.innerHTML = `<table><tbody>${tableData.join('')}</tbody></table>`;
+};
+
+// Form Submit
+const nameInput = document.querySelector('#name');
+const scoreInput = document.querySelector('#score');
+const formSubmitBtn = document.querySelector('#submit-btn');
```
Comment (rloterh) [inline src/modules/showScore.js]: - [ ] Unable to add a score detail due to errors thrown in the browser console. It appears the object `formSubmitBtn` cannot find a corresponding element in the DOM with the ID  `submit-btn`, hence the error. Please ensure to resolve the same. 

**Submit error:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mert, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦   🏃 

## Highlights 👍  
- Good job implementing API connection so far!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- [ ] OPTIONAL
Kindly include appropriate instructions under the _Getting Started_ section, especially on how to _run_ and/or _test_ your app.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/showScore.js (hunk 1 above)
Comment (mertkantaroglu) [reply src/modules/showScore.js]: Thanks @rloterh . It is interesting though. The program was working fine on my computer even with the typo. I just fixed it and submitted again.

**Review by rotimiazeez — APPROVED** [review]
Hi @mertkantaroglu 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter error ✔️
- Descriptive PR and README 👍🏽

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**